src/strace.c

```diff
@@ -870,7 +874,7 @@ printleader(struct tcb *tcp)
 	set_current_tcp(tcp);
 	current_tcp->curcol = 0;
 
-	if (print_pid_pfx || (nprocs > 1 && !outfname)) {
+	if (print_pid_pfx || (nprocs > 1 && !outfname) || always_show_pid) {
```

Comment:
This would make `--always-show-pid -ff -o` print pid prefix into each file using a format that's different from the format used to print pid prefix in case of `-f -o`.  Was this intended?

Reply:
Yes. But this behavior seems to contradict with the description in documentation. will be fixed in next push.